[2025-05-24T22:36:53.857+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-24T22:36:53.874+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_sql_data manual__2025-05-24T21:19:22.630212+00:00 [queued]>
[2025-05-24T22:36:53.881+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_sql_data manual__2025-05-24T21:19:22.630212+00:00 [queued]>
[2025-05-24T22:36:53.881+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 7
[2025-05-24T22:36:53.893+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): extract_sql_data> on 2025-05-24 21:19:22.630212+00:00
[2025-05-24T22:36:53.906+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1933) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-24T22:36:53.908+0000] {standard_task_runner.py:63} INFO - Started process 1935 to run task
[2025-05-24T22:36:53.908+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'supplier_data_etl_pipeline', 'extract_sql_data', 'manual__2025-05-24T21:19:22.630212+00:00', '--job-id', '166', '--raw', '--subdir', 'DAGS_FOLDER/supplier_dag.py', '--cfg-path', '/tmp/tmpsv68h8t1']
[2025-05-24T22:36:53.912+0000] {standard_task_runner.py:91} INFO - Job 166: Subtask extract_sql_data
[2025-05-24T22:36:53.966+0000] {task_command.py:426} INFO - Running <TaskInstance: supplier_data_etl_pipeline.extract_sql_data manual__2025-05-24T21:19:22.630212+00:00 [running]> on host a7d211f7b093
[2025-05-24T22:36:54.027+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='supplier_data_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract_sql_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-24T21:19:22.630212+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-24T21:19:22.630212+00:00'
[2025-05-24T22:36:54.028+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-24T22:36:54.030+0000] {etl.py:77} INFO - connected to db source
[2025-05-24T22:36:54.044+0000] {etl.py:79} INFO - queried db source
[2025-05-24T22:36:54.044+0000] {etl.py:81} INFO - closed db connection
[2025-05-24T22:36:54.046+0000] {etl.py:83} INFO - db source pickled
[2025-05-24T22:36:54.046+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-24T22:36:54.046+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-24T22:36:54.053+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=supplier_data_etl_pipeline, task_id=extract_sql_data, execution_date=20250524T211922, start_date=20250524T223653, end_date=20250524T223654
[2025-05-24T22:36:54.102+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-24T22:36:54.117+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-24T22:36:54.118+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
